src/core/Directus/Util/Installation/InstallerUtils.php

```diff
@@ -894,6 +894,7 @@ private static function createConfigData(array $data)
             'feedback_token' => sha1(gmdate('U') . StringUtils::randomString(32, false)),
             'feedback_login' => true,
             'timezone' => get_default_timezone(),
+            'logs_path' => __DIR__ . '/../../../../../logs',
```

Comment:
Will this use the directory of the installer utils or the config folder?

Reply:
It will use the installer utils directory

Reply:
It has to be the config directory though... 😬 otherwise the logs will be in /src/Directus/Util/Installation instead of /logs

Reply:
I meant `__DIR__` contain the utils directory and for converting it I used '/../../../../..'

So, basically, your logs will be at same place :) (`api/logs` folder)

Reply:
Perfect!